### ISSUE TITLE
ChoiceInput checkbox has aria-hidden error

### ISSUE THREAD

**Issue by roblevintennis** [AgnosticUI/agnosticui#215]
**Describe the bug**
ChoiceInput may be incorrectly using `aria-hidden`

**To Reproduce**
1. Run the playground application at playgrounds/SignupForm/src/components/react/React.jsx
2. I had done `npm run build && npm run preview` but likely `npm run dev` would work too
3. Apply Lighthouse to page

**Expected behavior**
No Lighthouse errors

**Screenshots**

Looks like we're getting a:
>[aria-hidden="true"] elements contain focusable descendants

![image](https://user-images.githubusercontent.com/142403/166246914-f1a73478-4d9f-4435-b474-523ea3d14efd.png)

**Comment by roblevintennis** [comment]
This should be fixed in all of the packages as of 5a80de0fb59df7963588e6b932bbd1489a32d927 -- I bumped package version numbers but haven't yet published these so the fixes are only in `master` not published npm packages (yet)
